Nyakaru/Bitcoin-Challenge
# Connect server and client

**What does this PR do?** 
Connect client with API data
**Description of Task to be completed?** 
Consume APIs
 **How should this be manually tested?** 
- Create a local copy `git fetch origin ch-connect-views` 
- Checkout to the branch `git checkout ch-connect-views`` 
- On running the app you should be able to access data from the api
